docs/config.rst

```diff
@@ -149,6 +149,14 @@ If you want to disable virtualenv creation altogether, you can set ``python`` to
     def tests(session):
         pass
 
+Use of :func:`session.install()` is deprecated without a virtualenv since it modifies the global Python enviroment. If this is what you really want, use :func:`session.run()` and pip instead.
```

Comment:
Our new `codespell` pre-commit hook found a typo here 💯 

```suggestion
Use of :func:`session.install()` is deprecated without a virtualenv since it modifies the global Python environment. If this is what you really want, use :func:`session.run()` and pip instead.
```

Reply:
Good stuff! 

Fixed 👍